packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -538,6 +539,12 @@ export interface CommonProjectProps {
    * @see https://docs.aws.amazon.com/codebuild/latest/userguide/test-report-group-naming.html
    */
   readonly grantReportGroupPermissions?: boolean;
+
+  /**
+   * Information about logs for the build project. A project can create logs in Amazon CloudWatch Logs, an S3 bucket, or both.
+   * @default - no log configuration is set
+   */
+  readonly logsConfig?: LogsConfig;
```

Comment:
Gotta say, I don't love the `logsConfig` name.

How about `logging`?

Reply:
I’m cool with it, I don’t love the name either.